examples/dreambooth/train_dreambooth.py

```diff
@@ -87,10 +87,9 @@ def save_model_card(
 # DreamBooth - {repo_id}
 
 This is a dreambooth model derived from {base_model}. The weights were trained on {prompt} using [DreamBooth](https://dreambooth.github.io/).
-You can find some example images in the following. \n
-{img_str}
+{f"You can find some example images in the following. \n\n{img_str}" if img_str != "" else ""}
 
-DreamBooth for the text encoder was enabled: {train_text_encoder}.
+{f"LoRA for the text encoder was enabled." if train_text_encoder else ""}
```

Comment:
This script doesn't have LoRA. 

```suggestion
{f"DreamBooth for the text encoder was enabled." if train_text_encoder else ""}
```

Reply:
This is in the dreambooth LoRA script, so I think it should stay there, but I can change it if you really want.